data/input-remapper.glade

```diff
@@ -873,12 +873,12 @@ Gives your keys back their original function</property>
                               </packing>
                             </child>
                             <child>
-                              <object class="GtkLabel">
+                              <object class="GtkComboBox" id="target-selector">
                                 <property name="visible">True</property>
                                 <property name="can-focus">False</property>
                               </object>
```

Comment:
Added tooltip, homogeneous positioning of buttons and dropdown: https://gist.github.com/sezanzeb/e542fc25deb2e1bb894959827abc1d22